src/styles/index.css

```diff
@@ -0,0 +1,154 @@
+.body {
+  margin: 0;
+}
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+}
+
+.button:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
+  color: #00acdc;
+}
+
+.card {
+  width: 199px;
+  height: 100%;
+  border: 1px solid rgb(240, 7, 228);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
```

Comment:
Styles for `.card` should be defined in `card.scss` and then compiled to CSS. This will maintain the benefits of using SCSS.